src/java.base/share/classes/jdk/internal/util/OctalDigits.java

```diff
@@ -58,15 +64,8 @@ final class OctalDigits implements Digits {
         DIGITS = digits;
     }
 
-    /**
-     * Constructor.
-     */
-    private OctalDigits() {
-    }
-
     @Override
-    public int digits(long value, byte[] buffer, int index,
-                      MethodHandle putCharMH) throws Throwable {
+    public int digits(long value, byte[] buffer, int index, MethodHandle putCharMH) throws Throwable {
```

Comment:
Can you revert these IDE reformatting changes? So that we don't pollute the changeset and the Git blame log, making reviews and reverting patches easier.

For example, only 3 lines of changes are truly necessary in this class:
Changing the package, `public ` before `final class OctalDigits` and `static final Digits INSTANCE`.